installinator/src/mock_peers.rs

```diff
@@ -441,19 +440,21 @@ mod tests {
         #[strategy((0..2000u64).prop_map(Duration::from_millis))]
         timeout: Duration,
     ) {
+        let log = test_setup_log("proptest_fetch_artifact");
```

Comment:
This returns a log context, and I think you need to call `logctx.cleanup_successful()` at the end of the test.

Reply:
Thanks, done.